discovery/zookeeper/zookeeper_test.go

```diff
@@ -0,0 +1,454 @@
+package zookeeper
+
+import (
+	"github.com/joyent/containerpilot/discovery"
+	"github.com/samuel/go-zookeeper/zk"
+	"testing"
+	"time"
+)
+
+// Factories, utilities
+func zkConnection() *zk.Conn {
+	c, _, _ := zk.Connect([]string{"127.0.0.1"}, time.Second)
```

Comment:
I’m trying to wrap my head around the Makefile etc. I can create the Docker instance but I’m getting a bit lost. Could you please give me some pointers about other files to tweak? That would be really helpful. 

I could also document it in the README, if you think it could be useful. Let me know.